lib/src/conflicts.rs

```diff
@@ -97,6 +99,16 @@ impl<T> Conflict<T> {
         }
     }
 
+    /// Returns the resolved value, if this conflict is resolved. Does not
+    /// resolve trivial conflicts.
+    pub fn into_resolved(self) -> Option<T> {
+        if self.removes.is_empty() {
+            self.adds.into_iter().next()
+        } else {
+            None
+        }
+    }
```

Comment:
nit: there's no caller of `into_resolved()`, and `-> Result<T, Self>` might be better so caller can retake ownership.

Reply:
> `-> Result<T, Self>` might be better so caller can retake ownership.

That's clever.